src/components/FeeDrawer.tsx

```diff
@@ -40,8 +36,8 @@ export default function FeeDrawer({
   feeHasError,
   totalFee,
   testID,
-  currencyInfo,
   showLocalAmount,
+  tokenId,
```

Comment:
Are `FeeDrawer` and `LegacyFeeDrawer` an example of where the "Legacy" version could be a wrapper over the new, `tokenId` version? e.g., `LegacyFeeDrawer` could continue to accept its current parameters, but internally, get a `tokenId` from the given `currency` and pass it through to `FeeDrawer`?

Reply:
I don't think so, `LegacyFeeDrawer` uses a weird `currency` param that is hard coded in it's use cases but to different values, and it also uses the (now) deprecated `CurrencyDisplay` component which does some weird stuff behind the scenes.